.gitignore

```diff
@@ -22,3 +22,5 @@ tilt_modules/
 
 # Jetbrains
 .idea/
+
+!vendor/
```

Comment:
Why do we now need to vendor dependencies?  Is it absolutely necessary? These don't look security critical but do we have a mechanism to ensure they get upgraded as either the vector version gets updated or we discover security flaws?